apps/zero-runtime-next-app/src/app/alert/page.tsx

```diff
@@ -0,0 +1,45 @@
+import { styled } from '@mui/zero-runtime';
+import BasicAlerts from '../../../../../docs/data/material/components/alert/BasicAlerts';
```

Comment:
At first, I tried to pull all the demos programmatically and render them with lazy loading but not successful, so I'll just go with importing manually and revisit this when I have more time.